corehq/tests/optimizer.py

```diff
@@ -122,6 +119,8 @@ def _extract_tests(test_suite_or_test):
 @nottest
 @contextmanager
 def optimize_apps_for_test_labels(test_labels):
+    # TODO make this work on Django 1.8 (django.test.simple has been removed)
```

Comment:
@czue / @dimagi/team-commcare-hq: Is this optimizer still valuable and used by people? It's disabled when using `REUSE_DB=1`, although can be enabled with [`REUSE_DB=optimize`](https://github.com/dimagi/commcare-hq/blob/e7d5b3e66029d60100c4e86ac8577963172e7cae/corehq/tests/nose.py#L287-L288) (FWIW, I've never used `REUSE_DB=optimize` except possibly while implementing it). If not, should we remove it rather than fixing this?